storage/ndb/src/rondis/src/rondb.cc

```diff
@@ -150,7 +242,7 @@ int rondb_redis_handler(const pink::RedisCmdArgsType &argv,
       return 0;
     }
     response->append("+PONG\r\n");
-  } else if (argv[0] == "ECHO") {
+  } else if (strcasecmp(command, "echo") == 0) {
```

Comment:
Perhaps you could optimize this by turning command into lowercase once and then do `strcmp`.

Reply:
Later, need a much more clever way of selecting command using hash, so no need to micro optimise before that